common/diagnostics.py

```diff
@@ -1,3 +1,11 @@
+# SPDX-FileCopyrightText: © 2022 Christian BUHTZ <c.buhtz@posteo.jp>
+# SPDX-FileCopyrightText: © 2022 Jürgen Altfeld <python@altfeld-im.de>
```

Comment:
I'd say `git blame` is totally on your side regarding the copyright (my contributions are not worth too be mentioned as copyright ;-)